Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeHidParsers.h

```diff
@@ -0,0 +1,119 @@
+#pragma once
+
+#include "GazeHidUsages.h"
+
+using namespace Windows::Devices::HumanInterfaceDevice;
+using namespace Windows::Devices::Input::Preview;
+
+BEGIN_NAMESPACE_GAZE_INPUT
+
+namespace GazeHidParsers {
+    public ref class GazeHidPosition sealed
```

Comment:
Is there a specific reason why these are not direct public properties instead of going through an indirection?

Reply:
There are various subtleties of the c++/cx projection, some of which cause odd errors and require specific implementations that aren't always intuitive.. I believe this is an artifact of one of those iterations. I tried the simplified version and it seems to work.